sites/public/pages/applications/review/demographics.tsx

```diff
@@ -24,15 +24,20 @@ import {
   PageView,
   pushGtmEvent,
   AuthContext,
+  listingSectionQuestions,
 } from "@bloom-housing/shared-helpers"
 import FormBackLink from "../../../src/forms/applications/FormBackLink"
 import { useFormConductor } from "../../../lib/hooks"
 import { UserStatus } from "../../../lib/constants"
+import { ApplicationSection } from "@bloom-housing/backend-core"
 
 const ApplicationDemographics = () => {
   const { profile } = useContext(AuthContext)
   const { conductor, application, listing } = useFormConductor("demographics")
-  const currentPageSection = 5
+  let currentPageSection = 4
+  if (listingSectionQuestions(listing, ApplicationSection.programs)?.length) currentPageSection += 1
+  if (listingSectionQuestions(listing, ApplicationSection.preferences)?.length)
+    currentPageSection += 1
```

Comment:
Is it possible to have this kind of logic in the ApplicationConductor, so we don't have to have it in multiple files?

Reply:
I would love to, it seems like there are a couple places we could choose to do that in ApplicationConductor, would you be okay splitting that into a separate ticket?

Reply:
Sounds good, @emilyjablonski. Here's the issue https://github.com/bloom-housing/bloom/issues/2972.